hive-event-listeners/apiary-gluesync-listener/src/main/java/com/expediagroup/apiary/extensions/gluesync/listener/ApiaryGlueSync.java

```diff
@@ -76,8 +82,45 @@ public ApiaryGlueSync(Configuration config, AWSGlue glueClient, String gluePrefi
   }
 
   @Override
-  public void onCreateTable(CreateTableEvent event) throws MetaException {
-    if (event.getStatus() == false) {
+  public void onCreateDatabase(CreateDatabaseEvent event) {
+    if (!event.getStatus()) {
+      return;
+    }
+    Database database = event.getDatabase();
+    try {
+      CreateDatabaseRequest createDatabaseRequest = new CreateDatabaseRequest()
+          .withDatabaseInput(transformDatabase(database));
+      glueClient.createDatabase(createDatabaseRequest);
+      log.debug(database + " database created in glue catalog");
+    } catch (AlreadyExistsException e) {
+      log.debug(database + " database already exists in glue, updating....");
```

Comment:
If it fails I think it would be better to log as an error rather than debug

Reply:
i think the debug logs in this class should actually be info...

Reply:
yeah not sure why debug was chosen maybe too much log messages but given databases are a rarity info is probably a good idea.